packages/bff/src/config/config.ts

```diff
@@ -67,10 +67,20 @@ export const AuthorizationProcessServerConfig = z
   .object({
     AUTHORIZATION_PROCESS_URL: APIEndpoint,
     TENANT_ALLOWED_ORIGINS: z.string(),
+    SAML_AUDIENCE: z.string(),
+    PAGOPA_TENANT_ID: z.string(),
+    SAML2_CALLBACK_URL: z.string().url(),
+    SAML2_CALLBACK_ERROR_URL: z.string().url(),
+    SUPPORT_LANDING_JWT_DURATION: z.coerce.number(),
```

Comment:
same, for compatibility this should be `SUPPORT_LANDING_TOKEN_DURATION_SECONDS`.
Plus this is optional in Scala, with a default of 300.

Note: as per other comment, this should probably be `SUPPORT_TOKEN_DURATION_SECONDS` (optional, with a default value of 3600)